docs/6.Contribution/Contribute New OpenAPI Policies.md

```diff
@@ -0,0 +1,150 @@
+                                                                                     ---
+layout: default
+published: true
+title: Contribute New OpenAPI configuration policy
+nav_order: 5
+---
+
+# Contribute New OpenAPI configuration policy
+
+In this example, we'll add support for a new OpenAPI configuration check.
+
+
+### Add a Check
+
+Go to `checkov/openapi/checks/resource`, go to v2|v3|generic according to the OpenAPI version you check and add `GlobalSecurityFieldIsEmpty.py`:
+
+v2 - OpenAPI 2.0.  
+v3 - OpenAPI 3.  
+generic - for both OpenAPI 2 and 3.
+
+```python
+class GlobalSecurityFieldIsEmpty(BaseOpenapiCheck):
```

Comment:
```suggestion
from __future__ import annotations

from typing import Any

from checkov.common.models.enums import CheckResult, CheckCategories
from checkov.common.checks.enums import BlockType
from checkov.openapi.checks.base_openapi_check import BaseOpenapiCheck


class GlobalSecurityFieldIsEmpty(BaseOpenapiCheck):
```
best to add the imports also in the example, otherwise newcomers who are not so familiar with Python will fail at this point.